ods_ci/tests/Tests/400__ods_dashboard/415__ods_dashboard_projects/415__ods_dashboard_projects.robot

```diff
@@ -219,7 +223,7 @@ Verify User Can Create A S3 Data Connection And Connect It To Workbenches
     ...                          aws_s3_endpoint=${DC_S3_ENDPOINT}    aws_region=${DC_S3_REGION}
     ...                          connected_workbench=${workbenches}
     Data Connection Should Be Listed    name=${DC_2_S3_NAME}    type=${DC_S3_TYPE}    connected_workbench=${workbenches}
-    Run Keyword And Continue On Failure    Workbench Status Should Be     workbench_title=${WORKBENCH_TITLE}
+    Run Keyword And Ignore Error    Wait Until Workbench Is Started     workbench_title=${WORKBENCH_TITLE}
```

Comment:
the old version was checking that the workbench is in "Starting" status. The new check is useless especially cos it's duplicated with the very next check

Reply:
I will remove the whole line, since as you said we're waiting for the workbench to start anyway on line 226.

Reply:
Yes, because sometimes more time passed between the workbench creation until this line, where the status already changed to running, and thus it failed (false failure).